coupon_chatter/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+#. Go to either *Sales > Products > Promotion Programs* or *Sales > Products > Coupon
```

Comment:
These instructions for use have changed:
![image](https://github.com/OCA/sale-promotion/assets/118818446/4cb112e3-b0e4-4002-93dc-3aaffc1a0d3d)

Reply:
Yes, that's true. Thanks for the notice.